app/api/v1/views/redflags.py

```diff
@@ -1,6 +1,6 @@
 from flask_restful import Resource
 from flask import jsonify, make_response, request
-from ..models.Redflags import RedFlagsModel, SingleFlagModel
+from ..models.Redflags import RedFlagsModel
 
 
 class RedFlags(Resource, RedFlagsModel):
```

Comment:
```suggestion
class RedFlags(Resource):
```
Why are you inhertiting models here?

Reply:
Also there is no point in overriding the `__init__` for a resource